### PR TITLE
[Instrumental] Underscore metric name output

### PR DESCRIPTION
**Pull request by janxious**
Some of the metrics emitted by various input plugins include illegal metric names for Instrumental, and this does a more thorough job of massaging those metrics into something usable.

### Required for all PRs:

- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [x] ~~README.md updated (if adding a new plugin)~~

### CODE REVIEW

**Comment by sparrc** [comment]
you need to run `go fmt ./...`

**Comment by janxious** [comment]
@sparrc It is done!

File: plugins/outputs/instrumental/instrumental.go [hunk 1]
```diff
@@ -131,8 +132,17 @@ func (i *Instrumental) Write(metrics []telegraf.Metric) error {
...
+			// decompose "metric.name value time"
+			splitStat := strings.SplitN(stat, " ", 3)
+			metric := splitStat[0]
+			value := splitStat[1]
+			time := splitStat[2]
+
+			// replace invalid components of metric name with underscore
+			clean_metric := MetricNameReplacer.ReplaceAllString(metric, "_")
+
+			if !ValueIncludesBadChar.MatchString(value) {
```
Comment (sparrc) [inline plugins/outputs/instrumental/instrumental.go]: do we still need to be checking this? how would it ever be true if we already replaced the bad characters above?

Comment (mediocretes) [inline plugins/outputs/instrumental/instrumental.go]: The above cleans up the metric name, this line rejects bad values.  If someone sends a name with bad characters, we just turn them to underscores, but if someone sends a bad value, we simply drop it.

I think the bad value that triggered this change was a time (as in HH:MM).

Comment (janxious) [inline plugins/outputs/instrumental/instrumental.go]: @sparrc bumps

**Comment by janxious** [comment]
Thanks @sparrc !

**Pull request merged**